### ISSUE TITLE
Themes polish

### ISSUE THREAD

**Issue by Klona13** [rr-/TRCustoms#312]
1. These should be the same:

Both highlight bg and fg colors should just follow the navbar active colors for all themes.
So...
![image](https://user-images.githubusercontent.com/93257509/178098976-13510d9d-764b-4597-b8ec-955fb489ae30.png)
Follows:
![image](https://user-images.githubusercontent.com/93257509/178098969-5dad1f60-b204-4799-b7ac-d844903cd657.png) 
So they should be grouped.

2. Some links aren't highlighted the same way.

It seems that all links use fg-color when they are "highlighted", except for the External links section and certain action links like (clear) (reset). I think they all should also use  --link-highlight-fg-color for consistency when the links are highlighted.

3. Set for these themes:

Sepia flashback
--link-highlight-fg-color: #eae0d3;

Diluted Calico
--link-highlight-fg-color: #eee5cd;

Metropolis
    --link-highlight-bg-color: #b9bdc6;
    --link-highlight-fg-color: #111145;

Sundown
--link-highlight-fg-color: #4e0404;

Film Noir
    --link-highlight-bg-color: #601c1c;
    --link-highlight-fg-color: #ffe9e9

**Comment by rr-** [comment]
@Klona13 
In the first point I'm told to group it with the navbar color, and in the third point I'm told to tweak these colors individually.

Currently we have a slight mess:
- Separate background and text color for the active navbar tab (`--navbar-active-tab-bg/fg-color`)
- Separate background and text color for the active page in the pager (`--link-highlight-bg/fg-color`)
- Separate background and text color for the buttons (`--button-bg/fg-color`)
- Hovered links happen to use `--link-highlight-fg-color` which may be a bad thing

Active page link kind of looks like a button, but it's not really a button.

For Metropolis reusing the active tab color for the active page link looks bad, since the pager appears on a white background whereas the active navbar tab appears on a grey background. So there is nuance in treating active page like an active navbar menu item.

Reusing `link-highlight-fg-color` which was meant for button-like links and that only makes sense on `link-highlight-bg-color`, to apply to hovered regular links that do not have backgrounds, is a conceptual mistake and should be ungrouped (it can still reference the original variable).

Having these in mind, LMK how to proceed, we can take a few iterations too if you want of grouping/ungrouping before we start to tweak individual colors.

**Comment by Klona13** [comment]
We can try what I suggested. I remember testing all these colors a while ago to make sure each individual theme was more consistent and it looked nice.

If you are wary about grouping, perhaps we can just change the colors so they're the same.

**Comment by rr-** [comment]
@Klona13 I pushed some changes that tried to make the color groupings more sensible, LMK your thoughts and if there are any color changes necessary.

**Comment by Klona13** [comment]
All themes use --fg-color: instead of --link-hover-color.

After that change, the colors suggested above should be in link hover color.

**Comment by rr-** [comment]
Not really, I wasn't sure what you meant. All links used fg-color which was very unintended.
